core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -706,6 +706,10 @@ oppia.directive('conversationSkin', [function() {
           return $scope.windowWidth < TWO_CARD_THRESHOLD_PX;
         };
 
+        $scope.isWindowTall = function() { 
+          return document.body.scrollHeight > window.innerHeight;
```

Comment:
Unfortunately it does not.

On Thu, May 26, 2016 at 12:37 PM, Allan Zhou <notifications@github.com>
wrote:

> In core/templates/dev/head/player/ConversationSkinDirective.js
> <https://github.com/oppia/oppia/pull/1882#discussion_r64807229>:
>
> > @@ -706,6 +706,10 @@ oppia.directive('conversationSkin', [function() {
> >            return $scope.windowWidth < TWO_CARD_THRESHOLD_PX;
> >          };
> >
> > +        $scope.isWindowTall = function() {
> > +          return document.body.scrollHeight > window.innerHeight;
>
> Does $document[0].body.scrollHeight work?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/oppia/oppia/pull/1882/files/6c9a1a46e967d4cea48415a0970fe940fa3ed6cc#r64807229>
>